tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -199,9 +214,15 @@ const openHamburgerMenu = async () => {
 };
 
 const closeHamburgerMenu = async () => {
+  if (await isHamburgerMenuClosed()) {
+    return;
+  }
+
   if (await isHamburgerMenuOpen()) {
-    await (await hamburgerMenu()).waitForClickable();
-    await (await hamburgerMenu()).click();
+    // await (await hamburgerMenu()).waitForClickable();
+    // await (await hamburgerMenu()).click();
+    await (await closeSideBarMenu()).waitForClickable();
+    await (await closeSideBarMenu()).click();
```

Comment:
Asking if the hamburger menu is open will also cover the scenario when it is closed because the method will not do anything if it is closed.

```js
const closeHamburgerMenu = async () => {
  if (await isHamburgerMenuOpen()) {
    await (await closeSideBarMenu()).waitForClickable();
    await (await closeSideBarMenu()).click();
  }
};

Reply:
This one still not resolved